pom.xml

```diff
@@ -65,7 +65,7 @@
     <compileSource>1.8</compileSource>
     <main.basedir>${project.basedir}</main.basedir>
     <ignore.test.failures>false</ignore.test.failures>
-    <jackson.version>2.10.2</jackson.version>
+    <jackson.version>2.14.1</jackson.version>
```

Comment:
It looks like 2.14.2 is the latest version. Is there a reason that you chose to go with 2.14.1 instead?

Reply:
I checked Nexus to see what the latest version we had there was, and up until I attempted a package with 2.14.2 just now, 2.14.1 was the latest version. 2.14.2 appears to work just fine, so I'll make that change.